src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -0,0 +1,118 @@
+import path from 'path'
+
+import { FormAuthType } from '../../../../shared/types'
+import config from '../../config/config'
+import { ControllerHandler } from '../core/core.types'
+import * as FormService from '../form/form.service'
+import * as PublicFormController from '../form/public-form/public-form.controller'
+import { RedirectParams } from '../form/public-form/public-form.types'
+import * as HomeController from '../home/home.controller'
+
+const RESPONDENT_COOKIE_NAME = 'v2-respondent-ui'
+const ADMIN_COOKIE_NAME = 'v2-admin-ui'
```

Comment:
Make them env-var so we can reset anytime

Reply:
https://github.com/opengovsg/FormSG/pull/3750/commits/2b2147ac222bf91ec70de5368f51ea2df56757c0